### PR TITLE
Add contact section

### LINKED ISSUE #22: Contact Section html of the web page
- [ ] Creates div elements for paragraph and map
- [ ]  Creates form elements for the contact
- [ ]  Uses h1 elements for the title and use p elements for the paragraph
- [ ]  Uses input, label, text-area and button elements for the contact form

### LINKED ISSUE #23: the Contact class

### PR DESCRIPTION
**Pull request by sabanyelkenci**

### CODE REVIEW

**Comment by sabanyelkenci** [comment]
hey @rago89, I tried to fix some line in my local for this PR and I think there is no error and there is no conflict right now. Could you please re-check and merge if it is okay for you or let me know what should I do.
thank you in advance

**Comment by snicoll** [comment]
@sabanyelkenci GitHub tells you whether there is a conflict right on this page, see the Green rectangle area below this comment.

**Comment by sabanyelkenci** [comment]
> @sabanyelkenci GitHub tells you whether there is a conflict right on this page, see the Green rectangle area below this comment.

Thanks @snicoll . If there are no conflict messages over there in this case  may I merge the PR or do we need to wait for the others review?

**Comment by snicoll** [comment]
GitHub telling you there is no conflict is not a review, it's a hint that the branch can be merged cleanly. It's important but a team member should look at your changes and provide you feedback. 

Think it this way. If you open a PR and the merge it yourself, why did you create a PR in the first place? You could just as well push that change directly to `main`.

**Comment by sabanyelkenci** [comment]
> GitHub telling you there is no conflict is not a review, it's a hint that the branch can be merged cleanly. It's important but a team member should look at your changes and provide you feedback.
> 
> Think it this way. If you open a PR and the merge it yourself, why did you create a PR in the first place? You could just as well push that change directly to `main`.

Yes @snicoll  you are right. In this case, I couldn't see any conflicts in this PR and it works in my local without any error. I want to know if there is anything should I do or not. Thanks a lot for your quick response and patient :

File: index.html [hunk 1]
```diff
@@ -58,6 +58,36 @@ <h1 class="showcase-h1">HELPING NEW TALENT INTO THE TECH WORLD</h1>
     <section class="our-partners"></section>
 
     <section class="support"></section>
+    
+    <section style="margin-bottom:50px" class="contact">
```
Comment (rago89) [inline index.html:76]: Know that this project is reverse engineering of HYF, but keep in mind that the code of the main page was made with a different type of code, as was explained in last Sunday class,  this software does not generate the best practices and semantical code, by example is not good practice having CSS code on the HTML, that's why we have classes.

**Review by rago89 — COMMENTED** [review]
Hi, see no errors, code is according to the HYF page, I left some suggestions to keep in main on future code, and a good way of improving the code from HYF page is to reverse but with the best practices

**Comment by rago89** [comment]
Good morning @sabanyelkenci can you please take a look and solve the merging conflicts

**Comment by sabanyelkenci** [comment]
> Good morning @sabanyelkenci can you please take a look and solve the merging conflicts

Hi @rago89  I solved the conflicts right now and I am going to merge this PR if everything OK for you

**Comment by snicoll** [comment]
@sabanyelkenci going forward, please use `git rebase` as we've discussed in class rather than a merge commit that is polluting the history unnecessary. 

(no need to do anything here as you've fixed the conflict already but I'd like you to try this technique for other PRs).

> Hi @rago89 I solved the conflicts right now and I am going to merge this PR if everything OK for you

Whenever a review is processed (like in this case, fixing the conflict) the reviewer (or another team member) should look at the changes again and validate them. Don't consider me as a team member.

**Comment by sabanyelkenci** [comment]
> @sabanyelkenci going forward, please use `git rebase` as we've discussed in class rather than a merge commit that is polluting the history unnecessary.
> 
> (no need to do anything here as you've fixed the conflict already but I'd like you to try this technique for other PRs).
> 
> > Hi @rago89 I solved the conflicts right now and I am going to merge this PR if everything OK for you
> 
> Whenever a review is processed (like in this case, fixing the conflict) the reviewer (or another team member) should look at the changes again and validate them. Don't consider me as a team member.

Thanks @snicoll  for your feedback. I hope I will be able to use this command the next time after I have more practised

**Comment by rago89** [comment]
Yes sure, try to do it according Sunday class

**Pull request merged**